CDCgov/prime-reportstream
# 12361 - removed activeMembership.service check where applicable since not all receivers have a service defined i.e."default"

This PR ...

Fixes the issue where the Data dashboard will not load if the reciever does not have a service defined in their active membership

Test Steps:
1. Log into RS as a receiver
2. Go to the Data Dashboard and verify that the page loads.
3. Go to the Facilities and Providers Page and verify that the page loads.
4. Log out
5. Log into RS as a sender
6. Make sure the Banner loads
7. Go to the Submissions page and verify that the page loads.
8. Go to the "Manage public key" page and verify that the page loads.

## Changes
![Screenshot 2023-11-27 at 1 16 39 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/c54ba953-27ff-40d5-b624-72af71346b0c)

![Screenshot 2023-11-27 at 1 16 46 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/d9e9bee8-b13e-4268-8bdb-d81d5628518c)

![Screenshot 2023-11-27 at 1 15 45 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/7f4a8dbe-9e7c-459a-b725-5c51c5a3b49b)

![Screenshot 2023-11-27 at 1 15 57 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/050727e6-b794-4ca3-a71d-71b5929cc131)

![Screenshot 2023-11-27 at 1 16 06 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/fad63b6f-d65b-4ce3-9048-a913ad92591e)


## Checklist

### Testing


## Linked Issues
- Fixes #12361
